packages/client/src/ActionsAndReducers/playerUi/helpers/handleVisibilityChanges.js

```diff
@@ -0,0 +1,30 @@
+import findAsset from '../../../Components/Mapping/helpers/findAsset'
+
+/** create a marker for the supplied set of details */
+export default (/* object */ message, /* object */ allForces) => {
+  const payload = message.payload
+  payload.forEach(visChange => {
+    const asset = findAsset(allForces, visChange.assetId)
+    if (!asset) {
+      console.warn('failed to find asset while handling vis id', visChange.assetId)
+    } else {
+      if (visChange.newVis) {
+        // ok, we're making something visible
```

Comment:
This sort of comment would be more useful if written with context and as an instruction, that way it is more obvious as to what it does as a glance.

Instead of writing `ok we're making something visible` write it as an imperative, e.g.: `Make [name of the thing rather than 'something'] visible to [who it is visible to]`

If [name of the thing] and [who it is visible to] then indicate that in the comment. e.g.

`Makes $asset visible to $force`

It's not massively important that you do this now but it's something I plan to enforce post-rebuild so we might as well start as we mean to go on.